### PR TITLE
WI benchmark data given by Steven and addressing review comments

### PR DESCRIPTION
**Pull request by bijujoseph**
@marimiyachi you can close: https://github.com/CMSgov/qpp-measures-data/pull/41
I have addressed the review comments in this one

### CODE REVIEW

**Comment by kencheeto** [comment]
@bijujoseph could you also commit the updated `benchmarks/2017.json` file?

**Comment by bijujoseph** [comment]
@kencheeto Good that you have asked me to checkin the measuredata json file. I noticed errors in validation step, but it is fixed now. Please re-review. Thank you

File: scripts/build-benchmarks [hunk 1]
```diff
@@ -3,5 +3,8 @@
 # generate benchmarks
 cat data/historical-benchmarks/2015.csv | node scripts/parse-benchmarks-data.js 2015 2017
 
+# merge the benchmark files
+echo "$(node scripts/merge-benchmarks.js ../benchmarks/2017.json ../data/wi_2017.json)" > benchmarks/2017.json
```
Comment (kencheeto) [inline scripts/build-benchmarks]: we can use [`jq`](https://stedolan.github.io/jq/) in lieu of the `merge-benchmarks` script below - jq is super handy when working with json:

```bash
echo "$(jq -s add benchmarks/2017.json data/wi_2017.json)" > benchmarks/2017.json
```

**Comment by bijujoseph** [comment]
Yes if you prefer `jq` I can do that. I was not sure if others have it installed on their dev boxes and on Travis CI. Please confirm...

**Comment by bijujoseph** [comment]
@kencheeto  modified to use `jq`

**Comment by kencheeto** [comment]
@bijujoseph nice! jq is already included in [CONTRIBUTING.md](CONTRIBUTING.md) and other scripts in this repo, and CI doesn't need it.

**Pull request merged**